### ISSUE TITLE
Sound buttons needs to be pressed twice

### ISSUE THREAD

**Issue by Alekx3** [rivafarabi/deckboard#85]
When I create a button to play a sound on the pc, I need to press 2 times from the app to make it work. The first time it wouldn't play, then the second it would play without issues.

**Comment by rivafarabi** [comment]
Noted. Thanks for pointing out.

**Comment by V3n0m15** [comment]
Could a delay option be added to all buttons? That way if this issue arises we can add a delay in order to solve the problem?

**Comment by spesvictus** [comment]
Sound buttons need to pushed twice because previous sound is not stopped.  You need to stop previous sound before new sound. 

I guess Deckboard needs to detect whether or not previous sound is still playing, if not then terminate

**Comment by Frensiow** [comment]
Yes the same thing happends to me, it feels like the button needs to be pressed to deactivate the sound before having to press it again or something. Very annoying.

**Comment by darkhog** [comment]
Year's almost up and it stil isn't fixed? At least, it isn't in the free version.
